frontend/pages/queries/QueryPage/QueryPage.jsx

```diff
@@ -329,18 +331,19 @@ export class QueryPage extends Component {
           });
       })
       .catch((campaignError) => {
-        if (campaignError === "resource already created") {
-          dispatch(
-            renderFlash(
-              "error",
-              "A campaign with the provided query text has already been created"
-            )
-          );
```

Comment:
@zwass In the interest of time, I routed all errors to `/500` until we've had a chance for a deeper look at error handling related to the `Fleet.entities` components